src/Libraries/PythonNodeModelsWpf/PythonIndentationStrategy.cs

```diff
@@ -0,0 +1,82 @@
+using ICSharpCode.AvalonEdit.Document;
+using ICSharpCode.AvalonEdit.Indentation;
+using ICSharpCode.AvalonEdit;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PythonNodeModelsWpf
+{
+    /// <summary>
+    /// Custom Indentation Strategy for Python 
+    /// https://csharp.hotexamples.com/site/file?hash=0x1abeea836b72c6db1910df1767e84d7bfcb620ed58a499d176fdf158851c1d5f&fullName=Yanitta/LuaIndentationStrategy.cs&project=Konctantin/Yanitta
+    /// </summary>
+    public class PythonIndentationStrategy : DefaultIndentationStrategy
```

Comment:
can this be made internal?

Reply:
Oh, yes, absolutely! Thank you for spotting that and the other one out, I will change them to internal.